src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -324,9 +325,7 @@ class ResourceService(private val resourceTypes: Map[ResourceTypeName, ResourceT
   }
 
   private def makeValidatablePolicies(policies: Map[AccessPolicyName, AccessPolicyMembership]): Future[Set[ValidatableAccessPolicy]] = {
-    Future.traverse(policies) {
-      case (accessPolicyName, accessPolicyMembership) => makeCreatablePolicy(accessPolicyName, accessPolicyMembership)
-    }.map(_.toSet)
+    policies.toList.traverse(policies => makeCreatablePolicy(policies._1, policies._2)).map(_.toSet)
```

Comment:
can you use a case statement so you can name policies._1 and policies._2?

Reply:
updated